### DISCUSSION TITLE
StandardControl and Control

### DISCUSSION THREAD

**Discussion by Niamh1** [maplibre/maplibre-gl-js#4170] [Q&A]
Hey! I when I add buttons on the map and run `npm run preview` they are all changing size or position.

If add the StandardControl to my map, and run `npm run dev`, I have the expected controls:
![Screenshot from 2024-05-24 17-47-52](https://github.com/maplibre/maplibre-gl-js/assets/3873524/ee2d2af4-6a19-4240-bf9d-1b70c219f58b)

If I then run `npm run build`, `npm run preview` (and when I deploy on firebase), the preview has the buttons the width of the map:
![Screenshot from 2024-05-24 17-47-20](https://github.com/maplibre/maplibre-gl-js/assets/3873524/455e2a6c-de9d-424c-9401-28d7c8a8164a)

I have also tried to create my own button, which appears in the top-right of the screen. Again it looks correct when running `npm run dev`, but moves to the left on `npm run preview`
`<Control position="top-right">
    <FilterButton/>
 </Control> `

Have you seen this behaviour before?

**Comment by HarelM** [comment]
The code described here is not part of maplibre-gl-js.
I suspect you are using some wrapper.
I advise to contact the wrapper owner...
